### PR TITLE
Rhc in master AB test fix

### PR DESCRIPTION
**Pull request by Calanthe**
Rhc sticky ad works correctly now on the Showcase articles.

### CODE REVIEW

**Comment by phamann** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js [hunk 1]
```diff
@@ -22,7 +26,7 @@ define([
         if (this.$adSlot.data('name') !== 'right') {
             return;
         }
-        articleBodyOffset = qwery('.content__article-body')[0].offsetTop;
+        articleBodyOffset = config.page.hasShowcaseMainPicture ? $('.media-primary').dim().height : qwery('.content__article-body')[0].offsetTop;
```
Comment (robertberry) [inline static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js]: This should probably be `fastdom`ed, as `dim()` and `offsetTop` will trigger a layout, and you're writing immediately after.

**Comment by robertberry** [comment]
Otherwise :+1:

**Comment by Calanthe** [comment]
The sticky mpu will be rewritten fully with fastdom later on. Ask @sndrs for more details

**Pull request merged**